### PR TITLE
[1 LOC] Fix broken URL

### PR DESCRIPTION
**Pull request by kgrhartlage**
## Description

- Update the URL to fetch data from the [1 LOC repo](https://github.com/phuocng/1loc)
- Upgrade deps
- Flag GitHub access token preference settings as required, because the extension does not work without it
- Update preference settings description with readable text, because URLs are not clickable in tooltips
- Add README to inform users about the need of a GitHub access token
- Update screenshots with new Raycast UI

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/1loc/src/index.tsx [hunk 1]
```diff
@@ -22,15 +23,17 @@ const useCategories = (): [ICategory[], boolean, AxiosResponse | null] => {
...
       const config = {
         cancelToken: cancelRef.current?.token,
         headers: personalAccessToken ? { Authorization: `token ${personalAccessToken}` } : undefined,
       }
       try {
-        const { data } = await axios.get(url, config)
+        const { data } = await axios.get<ICategory[]>(url, config)
         if (isMounted.current) {
-          setCategories(data)
+          const filtered = data.filter((item) => item.type === "dir" && !item.name.startsWith("_"))
```
Comment (kgrhartlage) [inline extensions/1loc/src/index.tsx:34]: The extension considers folders as categories, thus we need to filter out files. Additionally, folders named `_<name>` do not contain any snippets and should also be filtered out.

See their [project structure](https://github.com/phuocng/1loc/tree/master/contents) for reference.

**Comment by kgrhartlage** [comment]
> This pull request has been automatically marked as stale because it did not have any recent activity.
> 
> It will be closed if no further activity occurs in the next 7 days to keep our backlog clean 😊

I would need change requests for there to be any more activity from my side.

**Comment by kgrhartlage** [comment]
> @kgrhartlage is this ready to be merged from your side?
>

**Review by kud — APPROVED** [review]

**Comment by kud** [comment]
Thank you for the great job done here.

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**